setTRIMDAC_and_Mask.py

```diff
@@ -23,9 +25,9 @@
 
 for port in testSuite.presentVFAT2sSingle:
     try:
```

Comment:
I think the opening of this file can be done outside the VFAT loop.
That way you don't open and close this file 24 times